src/containers/TriggerBinding/TriggerBinding.scss

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+.triggerBinding {
+
+  h1 {
```

Comment:
will any of this be specific to TriggerBindings or should it be common across all pages?

Reply:
Not sure, but I believe so.

Reply:
if this is common we should move it somewhere else for reuse, and remove/rename references to `triggerBinding`

Reply:
Created an issue to follow up on this: https://github.com/tektoncd/dashboard/issues/844